### ISSUE TITLE
Add hyperketosis

### ISSUE THREAD

**Issue by jxchong** [obophenotype/human-phenotype-ontology#4245]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label

Hyperketosis

### 2. Synonyms

excess ketones

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

https://link.springer.com/article/10.1007/BF02072043
https://www.ncbi.nlm.nih.gov/pubmed/3189409
https://www.ncbi.nlm.nih.gov/pubmed/2303074

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))

HP:0001946

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)

diabetes

**Comment by pnrobinson** [comment]
Is there a distinction between the usage of ketosis and hyperketosis? Also, diabetes can be characterized by keotacidosis.
I am also wondering if we should not just use the term name "Hyperketonemia" (def: elevated level of ketone bodies in the blood), because Ketosis is defined as a metabolic state in which ketones are used for energy generation (this is not something we observe as a phenotype, it is an inference).

**Comment by LCCarmody** [comment]
Should hyperketosis be added as a synonym to ketosis?  We already have a term called 'ketonemia' which is defined as, "An increase in the level of ketone bodies in the blood.".  Should we add 'hyperketonemia; as a synonym to that term? @pnrobinson

**Comment by pnrobinson** [comment]
For now I think the best thing is to add hyperketonemia (primary label), synonym of ketonemia.
I think that for whatever reason, ketosis is used more than hyperketosis, which I will add as a synonym. I am thinking that ketosis or ketoacidosis is so commonly used in the clinics and the inference is so obvious that there is a good case to keep the terms.
